tests/probes/sysctl/test_sysctl_probe_all.sh

```diff
@@ -58,7 +58,14 @@ grep unix-sys:name "$result" | grep -v $SYSCTL_BLACKLIST_REGEX | sed -E 's;.*>(.
 # If procps_ver > 3.3.12 we need to filter *stable_secret and vm.stat_refresh
 # options from the sysctl output, for more details see
 # https://github.com/OpenSCAP/openscap/issues/1152.
-procps_ver=$(rpm -q procps-ng --qf="%{version}")
+if [ -f "/usr/bin/rpm" ]; then
```

Comment:
I'd propose creation of a function in `tests_commnon.sh`, like `fetch_package_version`. Could be useful for other distributions as well (like Arch) in the future.

Reply:
Good idea.  I can take a stab at that.